Remove options from the `test-title` rule (#224)
avajs/eslint-plugin-ava: docs/rules/test-title.md

Translations: [Français](https://github.com/avajs/ava-docs/blob/master/fr_FR/related/eslint-plugin-ava/docs/rules/test-title.md)

> Tests should have a title, because test without title would be rejected by AVA since [v1.0.1](https://github.com/avajs/ava/releases/tag/v1.0.1).

Reviewer: ```suggestion
Tests should have a title. AVA [v1.0.1](https://github.com/avajs/ava/releases/tag/v1.0.1) and later enforces this at runtime.
```
